apollo-http-cache/src/main/kotlin/com/apollographql/apollo3/cache/http/CachingHttpEngine.kt

```diff
@@ -21,7 +20,7 @@ class CachingHttpEngine(
     directory: File,
     maxSize: Long,
     fileSystem: FileSystem = FileSystem.SYSTEM,
-    private val delegate: HttpEngine = MultiplatformHttpEngine(),
+    private val delegate: HttpEngine = HttpEngine(),
```

Comment:
This reverts parts of https://github.com/apollographql/apollo-android/pull/3509. Since `HttpEngine` is a function, there's no need to prefix